detect.py

```diff
@@ -40,7 +40,12 @@ def detect(save_img=False):
     half = device.type != 'cpu'  # half precision only supported on CUDA
 
     # Load model
-    model = Darknet(cfg, imgsz).cuda()
+    # Arguman 'cpu' should be controlled, if 'cpu' is set model should be initialized with .cpu() method
+    # Older version of code cannot run without nvidia driver because of this reason
+    if device.type != 'cpu' 
```

Comment:
Add ":" and it works properly
```suggestion
    if device.type != 'cpu':
```